net/knxd/files/knxd.init

```diff
@@ -42,40 +44,50 @@ add_driver() {
 	add_str $cfg send_timeout
 	add_str $cfg debug
 	#ip
-	add_str $cfg multicast_address
-	add_str $cfg port
-	#ip #usb
-	add_str $cfg interface
-	#ipt #iptn #tpuarttcp #ncn5120tcp #ft12tcp #ft12cemitcp
-	add_str $cfg ip_address
-	add_str $cfg dest_port 
-	#ipt #iptn
-	add_str $cfg src_port
-	add_bool $cfg nat
-	add_str $cfg heartbeat_timer
-	add_str $cfg heartbeat_retries
-	add_str $cfg nat_ip
-	add_str $cfg data_port
-	#usb
-	add_str $cfg bus
-	#usb #tpuart(s) #ncn5120
-	add_str $cfg device
-	#usb
-	add_str $cfg config
-	add_str $cfg setting
-	add_str $cfg version
-	#usb
-	add_str $cfg bus
-	#usb #tpuart(s) #ncn5120
-	add_str $cfg device
-	#usb
-	add_str $cfg config
-	add_str $cfg setting
-	add_str $cfg version
-	#usb
-	add_str $cfg send_retries
-	#tpuart(s) #ncn5120 #ft12 #ft12cemi
-	add_str $cfg baudrate
+	if [ "$driver" == "ip" ] ; then
+		add_str $cfg multicast_address
+		add_str $cfg port
+	fi
+	if [ "$driver" == "ip" ] \
+	|| [ "$driver" == "usb" ] ; then
+		add_str $cfg interface
```

Comment:
I think using a [ ... ] || [ ... ] construct will fork a new sub-process for each [. So I think it would be better to use [ ... -o ... ]